google-cloud-clients/google-cloud-bigtable/README.md

```diff
@@ -9,8 +9,8 @@ heavy development and is not ready for production use. Please continue to use th
 [![Codacy Badge](https://api.codacy.com/project/badge/grade/9da006ad7c3a4fe1abd142e77c003917)](https://www.codacy.com/app/mziccard/google-cloud-java)
 
 - [Product Documentation][bigtable-product-docs]
-- [Client Library Documentation][bigtable-client-lib-docs]
-
+- [Client Library Documentation - Data API](https://googleapis.dev/java/google-cloud-clients/latest/com/google/cloud/bigtable/data/v2/package-summary.html)
+- [Client Library Documentation - Admin API](https://googleapis.dev/java/google-cloud-clients/latest/com/google/cloud/bigtable/admin/v2/package-summary.html)
 > Note: This client is under heavy development and should not be used in production.
```

Comment:
Should this go away?
Note: This client is under heavy development and should not be used in production.